invenio_app_rdm/records_ui/searchapp.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2021 CERN.
+#
+# Invenio-App-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Configuration helper for React-SearchKit."""
+
+from copy import deepcopy
+from functools import partial
+
+from flask import current_app
+from invenio_rdm_records.searchconfig import OptionsSelector, \
+    SortOptionsSelector
+
+
+class SortConfig(SortOptionsSelector):
+    """Sort options for the search configuration."""
+
+    def map_option(self, key, option):
```

Comment:
I was suspecting an override like this may happen...

Reply:
Yep, i've added a comment RDM-Records so that it's clear :-)